### PR TITLE
Add validatorUrl to the FastifySwaggerOptions interface

### PR DESCRIPTION
**Pull request by meeech**
No code changes. Just expose an option on the interface to make nice for us ts users.
satisfy #191 
#### Checklist

- [x ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included N/A
- [ ] documentation is changed or added N/A
- [x ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fastify-swagger",
-  "version": "4.8.3",
+  "version": "4.8.4",
```
Comment (mcollina) [inline package.json:3]: please revert this, we will update it when we release

**Review by mcollina — CHANGES_REQUESTED** [review]
Could you add a test for the types in https://github.com/fastify/fastify-swagger/tree/master/test/types?

File: package.json (hunk 1 above)
Comment (meeech) [reply package.json]: 👍🏼

**Comment by meeech** [comment]
> Could you add a test for the types in https://github.com/fastify/fastify-swagger/tree/master/test/types?

@mcollina absolutely - just will need some guidance, looking at the types test, this is the only one i guess it would fit in https://github.com/fastify/fastify-swagger/blob/master/test/types/types.test.ts#L129 just not clear what the test here is to validate things?

**Comment by mcollina** [comment]
Just adding the option to that file is fine.

**Comment by meeech** [comment]
> Just adding the option to that file is fine.

👍🏼

**Review by mcollina — APPROVED** [review]
lgtm

**Review by climba03003 — APPROVED** [review]
LGTM.

**Pull request merged**